tgui/packages/tgui/interfaces/PortablePump.js

```diff
@@ -0,0 +1,86 @@
+import { useBackend } from '../backend';
+import { Flex, Section, Button, NumberInput, LabeledList, Divider } from '../components';
+import { Window } from '../layouts';
+import { PortableBasicInfo, PortableHoldingTank } from './common/PortableAtmos';
+
+export const PortablePump = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    connected,
+    on,
+    direction_out,
+    holding,
+    pressure,
+    targetPressure,
+    maxPressure,
+    minRelease,
+    maxRelease,
+  } = data;
+
+  const handleTogglePower = () => {
```

Comment:
not sure why these need to be defined separately, they'd be clearer inline

Reply:
im assuming that means const thing = () => { act("thing"); } ?

Reply:
not exact but very close
look at code you already wrote
`onClick={() => handleSetPressure(maxRelease)}`
and then put it inline
`onClick={() => act("set-pressure", {maxRelease})}`

Reply:
will do

Reply:
did not work after much trying

Reply:
Did you look at how every other UI does it?

Reply:
i will try again

Reply:
done